python/nav/web/seeddb/utils/edit.py

```diff
@@ -99,9 +99,28 @@ def render_edit(
     }
     if obj:
         if obj.pk:
+            if verbose_name == "room":
+                detail_link = reverse_lazy('room-info', kwargs={'roomid': obj.pk})
+            elif verbose_name == "location":
+                detail_link = reverse_lazy(
+                    'location-info', kwargs={'locationid': obj.pk}
+                )
+            elif verbose_name == "vlan":
+                detail_link = reverse_lazy('vlan-details', kwargs={'vlanid': obj.pk})
+            elif verbose_name == "device group":
+                detail_link = reverse_lazy(
+                    'netbox-group-detail', kwargs={'groupid': obj.pk}
+                )
+            elif verbose_name == "prefix":
+                detail_link = reverse_lazy(
+                    'prefix-details', kwargs={'prefix_id': obj.pk}
+                )
+            else:
+                detail_link = ""
             context.update(
                 {
-                    'title': 'Edit %s "%s"' % (verbose_name, obj),
+                    'title': 'Edit %s' % verbose_name,
```

Comment:
This bit looks it inadvertently changes the appearance of SeedDB edit page titles. I'll have to spin it up on my desktop machine tomorrow to verify...

Reply:
It seems it does indeed make unwanted changes. Some examples will follow:

Reply:
Before this PR, this is what editing an organization looks like:
![Screenshot 2021-11-11 09 52 56](https://user-images.githubusercontent.com/100995/141279973-99f59346-f1ef-47f1-a727-34948c0e4c9b.png)

After, it looks like this:
![Screenshot 2021-11-11 09 53 42](https://user-images.githubusercontent.com/100995/141279969-a96c27ff-3d92-4e86-aa11-288f5dfb04e6.png)

The organization name is missing from the title.

Reply:
The same goes for editing e.g. Cabling.

Before the PR:

![Screenshot 2021-11-11 11 08 27](https://user-images.githubusercontent.com/100995/141280366-f17e5321-df52-4d7c-b661-a82dc8f148ed.png)

After the PR, cabling description is now gone from the edit form:

![Screenshot 2021-11-11 11 08 51](https://user-images.githubusercontent.com/100995/141280405-c75f059f-ab63-416a-9765-13d213171583.png)